### ISSUE TITLE
Terminate called after throwing an instance of 'std::runtime_error' on Linux

### ISSUE THREAD

**Issue by MattiaudaNicolasCN** [alexmercerind/dart_vlc#88]
Hi guys, referring to https://github.com/flutter/flutter/issues/87639 . When I try to run on **Linux** my app with your plugin installed I get:

 ```
Launching lib/main.dart on Linux in debug mode...
Building Linux application...
terminate called after throwing an instance of 'std::runtime_error'
Error connecting to the service protocol: failed to connect to http://127.0.0.1:3277
```
When I remove your plugin from the `.yaml` everything works correctly. 

Why does this happen?

**Comment by alexmercerind** [comment]
Anyways you should close issue at flutter repository.

**Comment by MattiaudaNicolasCN** [comment]
Yes, I closed it. 
But with the newer version of `dart_vlc_ffi` I continue to get the same error when I try to run my App

**Comment by alexmercerind** [comment]
Hey @MattiaudaNicolasCN,

I'm unable to experience the runtime crash. Anyways released a new version 0.1.1 with ton of memory leak fixes.

Keeping issue opened, provide details if you feel something is wrong. What other plugins are you using in your project?

![linux](https://user-images.githubusercontent.com/28951144/128404000-aed4638c-7f76-47d7-ac06-77f813159c51.png)
